index.src.html

```diff
@@ -727,8 +728,7 @@ <h4 id="cors-preflight">CORS preflight</h4>
                   to "`true`" in <var ignore>preflight</var>'s
                   [=request/header list=].
 
-      1.  Immediately after checking CORS-unsafe request-header names in
-          |response| against |request|'s [=request/header list=]:
+      1.  Immediately after the [=CORS check=]:
```

Comment:
```suggestion
      1.  Immediately after the [=CORS check=] succeeds:
```

Reply:
Done. FYI, I used the same version of bikeshed as you, as using the latest version of it introduced changes in index.html that were unrelated to my PR.